extensions/wikia/JSMessages/js/JSMessages.js

```diff
@@ -59,6 +59,14 @@
 			return ret;
 		};
 
+		JSMessages.html = function(){
+			var expandedText = this.msg.apply(this, arguments),
+				div = document.createElement('div');
+
+			div.appendChild(document.createTextNode(expandedText));
+			return div.innerHTML;
```

Comment:
There were a few approaches to escaping I found, most of which either listed out all the things to escape, or used jQuery or something similar to stuff content into the DOM and then remove it in a way that forces an escape of HTML (as does the above).  I chose this way because apparently it forces the browser do the escaping rather than JS so its much faster.